src/Components/Testimonials/TestimonialsForm.js

```diff
@@ -1,32 +1,96 @@
-import React, { useState } from 'react';
-import '../FormStyles.css';
+import {useEffect, useState} from 'react';
+import {Formik, Form, Field} from 'formik';
+import { ErrorsForm } from '../common/messagesForm';
+import { CKEditor } from '@ckeditor/ckeditor5-react';
+import { UrlInput } from '../common/getUrlInputFile';
+import ClassicEditor from '@ckeditor/ckeditor5-build-classic';
+import { validateFormTestimonials, replaceCKeditor } from '../common/validateFormTestimonials';
+import {getTestimonial,createTestimonial,modifiedTestimonial} from '../../Services/testimonialService';
+const TestimonialForm = ({testimonialId}) => {
 
-const TestimonialForm = () => {
-    const [initialValues, setInitialValues] = useState({
-       name: '',
-       description: '' 
-    });
+  const [imgState,setImgState] = useState();
+  const [dataForm, setDataForm] = useState({
```

Comment:
the name of this should be more representative of what it is, maybe just testimonial its a good candidate
```
const [testimonial, setTestimonial] = useState({...code here...});
```